### ISSUE TITLE
Non-void function does not return a value in all control paths

### ISSUE THREAD

**Issue by dimitre** [astellato/ofxSyphon#48]
This function returns this error
```
Non-void function does not return a value in all control paths
```

https://github.com/astellato/ofxSyphon/blob/01345475bd8dd14f0b14d3d32e39507953d95282/src/ofxSyphonClient.mm#L43-L63

**Comment by dimitre** [comment]
not pretty but can be solved with
```c++
     [pool drain]; 
     return *this; 
```

**Comment by ofTheo** [comment]
think this was closed by #49
